packages/security/src/index.ts

```diff
@@ -0,0 +1,7 @@
+declare var IRoot: any;
+
+export default class SecurityService {
+  constructor() {}
+
+  public isRooted = IRoot.isRooted;
```

Comment:
is it possible that `IRoot` is undefined?

Reply:
Yeah. This code should use the same aproach as in core where we:

1) Detect platform
2) Detect if plugin is available.